examples/metalk8s-solution-example/operator/pkg/config/config.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"gopkg.in/yaml.v2"
+)
+
+type TOperatorConfig struct {
+        Kind         string `yaml:"kind"`
+        APIVersion   string `yaml:"apiVersion"`
+        Repositories []struct {
+                Version    string `yaml:"version"`
+                Repository string `yaml:"repository"`
+        } `yaml:"repositories"`
+}
+
+var OperatorConfig TOperatorConfig
```

Comment:
Not sure some global mutable config 'singleton' is a great pattern.

Reply:
Changed this a little bit.